src/compas_timber/ghpython/components/CT_ShowAssembly/code.py

```diff
@@ -12,6 +12,6 @@ def RunScript(self, Assembly):
 
         Brep = []
         for beam in Assembly.beams:
-            Brep.append(Artist(beam.get_geometry(True)).draw())
+            Brep.append(Artist(beam.geometry).draw())
```

Comment:
check also other GH components, there are some more `beam.get_geometry` calls

Reply:
Nice catch :) Fixed in Bake component. Seen anywhere else?